numbthry/gaussInteger-s4478632/README.md

```diff
@@ -0,0 +1,19 @@
+# GaussInteger

```

Comment:
Good readme, but requires dependencies (even if only Python version) and usage example